rules/aws/exfiltration_rds_snapshot_export.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the export of an Amazon Relational Database Service (RDS) Aurora database snapshot.
+"""
+false_positives = [
+    """
+    Export Snapshots may be done by a system or network administrator. 
```

Comment:
```suggestion
        Exporting snapshots may be done by a system or network administrator. Verify whether the user
    identity, user agent, and/or hostname should be making changes in your environment. Snapshot exports from unfamiliar
    users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the
    rule.
```
re-lint